service/history/api/respondworkflowtaskfailedrequest/api.go

```diff
@@ -0,0 +1,111 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package respondworkflowtaskfailedrequest
```

Comment:
```suggestion
package respondworkflowtaskfailed
```

Reply:
Actually, why "respond"? Should it be simply `workflowtaskfailed`?

```
temporal(main અ ) ls service/history/api
activity_util.go              get_history_util.go                 listtasks                     removesignalmutablestate      retry_util.go            update_workflow_util.go
addtasks                      get_workflow_util.go                multioperation                replication                   signal_workflow_util.go  updateworkflow
consistency_checker.go        getdlqtasks                         namespace.go                  replication_util.go           signalwithstartworkflow  verifychildworkflowcompletionrecorded
consistency_checker_test.go   getworkflowexecutionhistory         pollupdate                    replicationadmin              signalworkflow           workflow_id_dedup.go
create_workflow_util.go       getworkflowexecutionhistoryreverse  queryworkflow                 requestcancelworkflow         startworkflow            workflow_lease.go
deletedlqtasks                getworkflowexecutionrawhistory      reapplyevents                 resetstickytaskqueue          task_category.go
deleteworkflow                getworkflowexecutionrawhistoryv2    recordactivitytaskheartbeat   resetworkflow                 task_category_test.go
describemutablestate          isactivitytaskvalid                 recordactivitytaskstarted     respondactivitytaskcanceled   terminateworkflow
describeworkflow              isworkflowtaskvalid                 recordchildworkflowcompleted  respondactivitytaskcompleted  token.go
forcedeleteworkflowexecution  listqueues                          refreshworkflow               respondactivitytaskfailed     trim_history_util.go
```

Reply:
Yup, 100% copy-paste; including the package name ;) Fixed!